src/config.js

```diff
@@ -17,6 +18,9 @@ const DefaultConfig = {
     maxDialsPerPeer: Constants.MAX_PER_PEER_DIALS,
     dialTimeout: Constants.DIAL_TIMEOUT
   },
+  keychain: {
+    datastore: new MemoryDatastore()
+  },
```

Comment:
This is going to cause us to carry around the MemoryDatastore even if we don't use the keychain. I think we should just default this to null and check the datastore before creating the keychain. There's no real value in a memory datastore for the keychain.

Reply:
you are right! just removed it and added a validation. In the docs it is already defined as required